src/editor.ts

```diff
@@ -181,6 +193,9 @@ export const yesterday: IMXIcon = {
 };
 
 export const editor = [
+  all,
```

Comment:
That's what's causing the issue right now. Please rename this to something more descriptive.

Reply:
Now should be resolved